.github/pull_request_template.md

```diff
@@ -1,4 +1,8 @@
 
 - [ ] I updated the README.md docs to reflect this change.
-- [ ] This is not a breaking API change OR 
-- [ ] This is a breaking API change and I have incremented the API version.
+
+For changes to the codebase:
+
+- [ ] I have written tests to cover this change.
+- [ ] This is not a breaking API change OR
+- [ ] This is a breaking API change and I have incremented the API version and added a summary to CHANGELOG.md.
```

Comment:
Some thoughts related to this:

I think we could add an entry in the changelog for nearly every PR.  The version we're using is really for the API, I suppose, and it might be worth having a separate version for the API vs. the spec, as they're two different release processes.